libs/cluster/Server/Replication/PrimaryOps/PrimarySendCheckpoint.cs

```diff
@@ -27,7 +27,7 @@ public ReadOnlySpan<byte> BeginReplicaSyncSession(string remoteNodeId, string re
             {
                 var message = "-PRIMARY-ERR: failed creating replica sync session task.\r\n";
                 logger?.LogError(message);
-                return Encoding.ASCII.GetBytes(message);
+                return "-PRIMARY-ERR: failed creating replica sync session task.\r\n"u8;
```

Comment:
Well, we could do `logger?.LogError(Encoding.ASCII.GetString(errorMessageRVASpan));`, having log call here means the perf isn't that big concern anyways 😄

Reply:
Or well, it would become akward, it'd require couple hardcoded slices to get the actual message.. I can restore original logic to not repeat the message here too.

Reply:
It should be fine as is. Ideally, we need to move the message into CmdStrings and then do GetString for the logger. Will get to it into separate PR.